tests/yaml/YAMLLintBearTest.py

```diff
@@ -83,3 +90,23 @@
                                       valid_files=(with_start_yaml_file,),
                                       invalid_files=(no_start_yaml_file,),
                                       settings={'document_start': True})
+
+
+@generate_skip_decorator(YAMLLintBear)
+class YAMLLintBearSeverityTest(unittest.TestCase):
+
+    def setUp(self):
+        self.section = Section('')
+        self.uut = YAMLLintBear(self.section, Queue())
+
+    def test_warning(self):
+        content = test_file.splitlines()
+        with prepare_file(content, None) as (file, fname):
+            with execute_bear(self.uut, fname, file) as results:
+                self.assertEqual(results[6].severity, RESULT_SEVERITY.NORMAL)
```

Comment:
@NiklasMM I have checked for the `result.message` now which will make clear about which messages are `error` and which are `warning`. I think using additional files will unnecessarily elongate the code.